analysis/spectra_chisq/_model_spectra.py

```diff
@@ -0,0 +1,261 @@
+"""The ``_model_spectra`` module models spectra by varying the dust extinction from the Milky Way.
+
+The model used is ``sncosmo.CCM89Dust``. The parameters A_v and amplitude are fit to the observed spectra. The best fit  parameters are determined by ``scipy.optimize.curve_fit``
+The fit parameters are saved to a fits file.
+"""
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.table import Table
+
+from functools import partial
+from scipy.optimize import curve_fit
+
+import sncosmo
+import sndata
+from sndata.csp import DR1, DR3
+
+
+dr1, dr3 = DR1(), DR3()
+dr1.download_module_data()
+dr3.download_module_data()
+dr3.register_filters(force=True)
+
+
+def jd_to_mjd(date):
+    """Convert JD date to MJD
+    
+    Args:
+        date (float): JD date
+        
+    Returns:
+        (float) MJD date
+    """
+    
+    return date + 2400000.0
+
+
+def get_t0(obj_id, tbl=dr3.load_table(3), mjd=False):
```

Comment:
When specifying default arguments, always try to use non-mutable values. Your default value for `tbl` is an astropy table which is a mutable object type. This means if the function is called with default arguments and the `tbl` argument is modified as the function runs, then the default value will change as well. Consider specifying the table number `3` as the default and load the table inside the function.